fvm/evm/emulator/emulator.go

```diff
@@ -270,6 +298,24 @@ func (proc *procedure) withdrawFrom(
 	return res, proc.commitAndFinalize()
 }
 
+func (proc *procedure) dryRun(call *types.DirectCall) (*types.Result, error) {
+
+	// run the procedure, hash and index don't matter since it won't be emitted anywhere
+	res, err := proc.run(
+		call.Message(),
+		gethCommon.Hash{},
+		0,
+		types.DryRunSubType,
```

Comment:
we could use unknown subtype

Reply:
this is actually not needed anymore, sorry forgot to delete left over code